test/ufuzz.js

```diff
@@ -183,43 +284,57 @@ function log() {
     console.log(beautify_result);
     console.log("uglified result:");
     console.log(uglify_result);
+    if (!ok) console.log("!!!!!! Failed...");
 }
 
 var num_iterations = +process.argv[2] || 1/0;
-var verbose = !!process.argv[3];
+var verbose = process.argv[3] === 'v' || process.argv[2] === 'v';
+var verbose_interval = process.argv[3] === 'V' || process.argv[2] === 'V';
```

Comment:
Yeah that's why I kept that the way it is. If the cli flags extend we should look to a more classic approach here, there are plenty of proper cli args npm packages out there we could use for this.

Fwiw, I like seeing the output every now and then, which is why I've added `V`. Especially when working on the fuzzer I want to review the running fuzzer, like to make sure it doesn't bias empty functions or something like that.